modules/openy_gc_log/src/LogArchiver.php

```diff
@@ -413,19 +420,19 @@ protected function prepareLogsForExport($logFileName = '') {
             $metadata = unserialize($log->get('event_metadata')->value, ['allowed_classes' => FALSE]);
           }
           foreach ([
-            'entity_title',
-            'entity_instructor_name',
-            'entity_created',
-          ] as $key) {
+            'entity_title' => 'Content title',
```

Comment:
t() ?

Reply:
thx. We will fix it right after release. Noot a release blocker.